agents/round.py

```
@@ -0,0 +1,146 @@
+                # TODO: claim stuff
+                # TODO: WON't Work!
+                agent_wallet = Wallet.fetch(wallet_id=agent.address)
+                # NOT TESTED
+                nounce = w3.eth.get_transaction_count(agent.address)
+                # tx = prediction_contract.functions.claim(claim_epoch).buildTransaction({
+                #     'from': agent.address,
+                #     'nonce': nounce,
+                # })
+                tx_hash = agent_wallet.transact(
+                    {
+                        "to": prediction_contract_address,
```

Ruff (0.8.2)

Undefined name `prediction_contract_address`

(F821)